components/switch/metadata/switch.yml

```diff
@@ -6,20 +6,43 @@ sections:
       This component can be modified via its `--mod-*` prefixed custom properties. A list of those prefixed custom properties can be found <a class="spectrum-Link" href="https://github.com/adobe/spectrum-css/tree/main/components/switch/metadata/mods.md">here</a>.
   - name: Migration Guide
     description: |
-      ### Component renamed
-      Toggle is now known as Switch. Replace all `.spectrum-ToggleSwitch*` classnames with `.spectrum-Switch*`.
-      ### T-shirt sizing
-      Switch now supports t-shirt sizing and requires that you specify the size by adding a .spectrum-Switch--size* class. The default size is "medium".
-      ### Quiet and emphasized
-      Spectrum has chosen the variant previously known as `quiet` to be the default and has added an `emphasized` variant with the same styles as the previous default.
-      If you were using the `quiet` variant, the `.spectrum-Switch--quiet` class is no longer required and can be removed.
-      If you need a switch to look like it did before this change, add `.spectrum-Switch--emphasized`.
```

Comment:
Are we okay with losing this history or could it be put under the old version number(s)?

Reply:
I was torn. I removed them because they're all five years old